.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
What if an RP belongs to two arcs at once though? (This is the direction I'm leaning towards with regards to multiple canons -- having a canon just be a big arc holding all the RPs that belong to it.)